client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -548,6 +553,7 @@ class PlansSetup extends React.Component {
 
 		return (
 			<div className="jetpack-plugins-setup">
+				{ this.renderPageViewTracker() }
```

Comment:
The function isn't doing anything interesting. Let's inline the component call here.

Reply:
Updated in 152380974e1fe5e4edc33de55a46102262327ecb.